docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/biography/node--cgov-biography--full.html.twig

```diff
@@ -1,111 +1,124 @@
-              {% endif %}
-              {% if content.field_org_name_2 %}
-              <br />
+            <div class="profile-panel-content">
+
+              <h1>
+                {% spaceless %}
+                  {% if content.field_prefix|field_value %}
+                    {{ content.field_prefix|field_value }}
+                  {% endif %}
+                  {{ content.field_first_name|field_value }}
+                  {% if content.field_middle_name %}
+                    {{ content.field_middle_name|field_value }}
+                  {% endif %}
+                  {{- content.field_last_name|field_value -}}
+                  {%- if content.field_suffix -%}
+                    ,&nbsp;{{ content.field_suffix|field_value }}
+                  {%- endif -%}
+                {% endspaceless %}
+              </h1>
+
+              {% if content.field_title|field_value %}
+                <div class="profile-title">{{ content.field_title|field_value }}</div>
               {% endif %}
-              {% if content.field_org_url_2 %}
-              <a href="{{ content.field_org_url_2 }}">
+
+              {% if content.field_org_name_1|field_value %}
+              <p>
+                <a href="{{ content.field_org_url_1|field_value }}" class="profile-org">{{ content.field_org_name_1 }}</a>
+                {% if content.field_org_name_2|field_value %}
+                  <a href="{{ content.field_org_url_2|field_value }}" class="profile-org">{{ content.field_org_name_2 }}</a>
+                {% endif %}
+              </p>
               {% endif %}
-              {% if content.field_org_name_2 %}
-              {{ content.field_org_name_2 }}
+                
+
+              {% if content.field_campus|field_value or content.field_room_number|field_value %}
+              <p>
+                {% if content.field_campus|field_value %}
+                  {{ content.field_campus }}
+                {% endif %}
+
+                {% if content.field_campus|field_value and content.field_room_number|field_value %}
+                  |
+                {% endif %}
+
+                {% if content.field_room_number|field_value %}
+                  {{ content.field_room_number }}
+                {% endif %}
+              </p>
               {% endif %}
-              {% if content.field_org_url_2 %}
-              </a>
+
+
+              {% if content.field_phone_number|field_value or content.field_email_address|field_value %}
+              <div class="profile-contact-info">
+                {% if content.field_phone_number|field_value %}
+                  <div class="profile-contact-phone">{{ content.field_phone_number|field_value }}</div>
+                {% endif %}
+                {% if content.field_phone_number|field_value and content.field_email_address|field_value %}
+                  <span class="contact-divider">|</span>
+                {% endif %}
+                {% if content.field_email_address|field_value %}
```

Comment:
I think field_email_address would be best handled as a field template. Since this markup does not have any bearing on any other fields, more of the Drupal theming subsystem can handle the checks for existence. Also in a field template you will have access to the value of the field, and not require twig field value.